src/backend/db/products.js

```diff
@@ -90,7 +90,7 @@ export const products = [
     title: "FERARI PILOTA",
     price: "600",
     mrpPrice: "1200",
-    discount: "50%",
+    discount: "50% off",
     rating: 3,
```

Comment:
you can calculate discount directly from mrp and price, so no need of this

Reply:
right i will do this right now